### PR TITLE
update changelogs and bump uint

### PR DESCRIPTION
**Pull request by ordian**
Should go after #486.

Can publish on crates.io finally when merged.

### CODE REVIEW

File: contract-address/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ The format is based on [Keep a Changelog].
 ### Breaking
 - Updated `ethereum-types` to 0.10. [#463](https://github.com/paritytech/parity-common/pull/463)
 
-## [0.9.0] - 2020-03-16
+## [0.4.0] - 2020-03-16
```
Comment (dvdplm) [inline contract-address/CHANGELOG.md:11]: Copy&paste mistake?

**Review by dvdplm — APPROVED** [review]

**Comment by dvdplm** [comment]
@ordian In the descr you reference this pull request, #486. Not sure which one you meant.

**Comment by dvdplm** [comment]
```
…
4b6bc9e29877: Downloading     170B/170B
[14B/usr/bin/docker: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
See '/usr/bin/docker run --help'.
travis_time:end:0aed4040:start=1608932447584171980,finish=1608932552036002448,duration=104451830468,event=script
[0K[31;1mThe command "cross test --target=aarch64-linux-android -p parity-util-mem" exited with 125.[0m
```

Travis is having issue. :(

File: contract-address/CHANGELOG.md (hunk 1 above)
Comment (ordian) [reply contract-address/CHANGELOG.md:11]: yep

**Comment by ordian** [comment]
> @ordian In the descr you reference this pull request, #486. Not sure which one you meant.

I meant #483, which is merged now.
> Travis is having issues. :(

yeah, that's unrelated to the PR though, looks like we've reached some docker limits for tests an android

**Review by niklasad1 — APPROVED** [review]

**Comment by ordian** [comment]
Would be good to have #496 and #488 (blocked on quickcheck?) in as well.

**Comment by dvdplm** [comment]
>  #488 (blocked on quickcheck?) in as well.

Yeah, afaict we can't upgrade before the next `quickcheck` release (or ditch `quickcheck`, which I'd rather not do).

**Comment by ordian** [comment]
> > #488 (blocked on quickcheck?) in as well.
> 
> Yeah, afaict we can't upgrade before the next `quickcheck` release (or ditch `quickcheck`, which I'd rather not do).

I don't think anyone uses quickcheck feature of `uint`, AFAIK it's only used for tests. We expose `arbitrary` feature for public use.
A way forward would be to make quickcheck dev-dependency (and move tests) and depend on some git revision, but I'd rather wait for a release.

**Comment by ordian** [comment]
@HCastano we're planning to publish a bunch of crates from this branch, including parity-crypto with https://github.com/paritytech/parity-common/pull/496, probably on 5th of January. Would appreciate if you could test or review the branch before we do this.

**Comment by ordian** [comment]
It just happened :)
https://github.com/paritytech/parity-common/releases

**Pull request merged**